product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -14,7 +14,7 @@ Different types of events are sent as notifications. These notifications are set
 | Project      | Upcoming maintenance upgrade on a cluster (24hr)                                                 | Project owner/editor             | All                  |
```

Comment:
What does "24hr" in parens mean? I understand it means 24 hour, but I does it mean (in 24 hours)? If so, I will change it to that.